src/store/reviewSpot/index.js

```diff
@@ -0,0 +1,179 @@
+import { getSpotRequestStatusLabel as getSpotStatus } from "../../constant/enums";
+import { getParkingSizeLabel as getParkingSize } from "../../constant/enums";
+import { getRentUnitLabel as getRentUnit } from "../../constant/enums";
+import { getSiteTypeLabel as getSiteType } from "../../constant/enums";
+import { mayaClient } from '@/services/api';
+
+const state = {
+    formdataSO: {
+        userName: '',
+        spotId: null,
+        fullName: '',
+        mobile: '',
+        email: '',
+        address: '',
+        city: '',
+        area: '',
+        latitude: null,
+        longitude: null,
+      },
+      formdataRent: {
+        totalSlots: null,
+        baseAmount: null,
+        rentUnit: '',
+        parkingSize: '',
+        siteType: '',
+      },
+      formdataBooking: {
+        startDate: '',
+        endDate: '',
+        lastCallDate: '',
+        duration: '',
+        spotrequestStatus: '',
+        remark: '',
+        },
+    mobileError: '',
+    latitudeError: '',
+    longitudeError: '',
+    hasError: false,
+    errorMessage: '',
+    isLoading: false,
+};
+const mutations = {
+    'set-error'(state, { field, message }) {
+        state[field] = message;
+    },
+    'set-global-error'(state, errorMessage) {
+        state.hasError = true;
+        state.errorMessage = errorMessage;
+    },
+    'reset-global-Error'(state) {
+        state.hasError = false;
+        state.errorMessage = '';
+    },
+    'set-loading'(state, isLoading) {
+        state.isLoading = isLoading;
+    },
+    'setFormData'(state, formData) {
+        state.formdataSO = { ...state.formdataSO, ...formData.SO };
+        state.formdataRent = { ...state.formdataRent, ...formData.Rent };
+        state.formdataBooking = { ...state.formdataBooking, ...formData.Booking };
+    },
+    'update_latitude'(state, latitude) {
+        state.formdataSO.latitude = latitude;
+    },
+    'update_longitude'(state, longitude) {
+        state.formdataSO.longitude = longitude;
+    },
+};
+
+const actions = {
+
+    // Validate Latitude type
+    validateLatitude({ commit, state }) {
```

Comment:
validateLatitude and validateLongitude are very similar, i feel we can extract a common function.